app/frontend/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -150,10 +152,19 @@ export default function GroupChatView() {
                     Chat members
                   </MenuItem>
                   {isChatAdmin && (
-                    <MenuItem onClick={() => handleClick("settings")}>
-                      Chat settings
+                    <MenuItem onClick={() => handleClick("admins")}>
+                      Manage chat admins
                     </MenuItem>
                   )}
+
+                  {
+                    //<Menu> doesn't accept JSX Fragment children
```

Comment:
Really? Seems odd that they have `children` typed as `React.ReactNode` which includes Fragment then 🤔

Reply:
It lints and works fine but gave console warnings